.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  root: true,
+  extends: ["airbnb-base", "prettier"],
+  env: {
+    node: true,
+    es6: true,
+    jest: true,
+  },
+  plugins: ["prettier"],
+  rules: {
+    "prettier/prettier": "error",
+    "one-var": 0,
+    "one-var-declaration-per-line": 0,
+    "new-cap": 0,
+    "consistent-return": 0,
+    "no-param-reassign": 0,
+    "comma-dangle": 0,
+    "no-console": 0,
+    "no-unused-vars": 0,
```

Comment:
Enable this rule

Reply:
I changed 0 to 1, they give warnings, Is that fine?